service/controller/resource/alert/rules/heartbeat.go

```diff
@@ -0,0 +1,25 @@
+package rules
+
+import (
+	promv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func Heartbeat(obj interface{}) promv1.RuleGroup {
+	return promv1.RuleGroup{
+		Name: "heartbeat",
+		Rules: []promv1.Rule{
+			promv1.Rule{
+				Alert: "Heartbeat",
+				Expr:  intstr.FromString(`time() - max(container_start_time_seconds{cluster_type="host", namespace="monitoring", container="prometheus"}) > 60`),
```

Comment:
Are you creating heartbeats as they exist currently here?

Reply:
please don't look at this PR :see_no_evil: 

There will be some changes to Heartbeats alert to route them correctly.